flake.nix

```diff
@@ -105,6 +105,44 @@
             vendorSha256 = "sha256-f/DIAw8XWb1osfXAJ/ZKsB0sOmFnJincAQlfVHqElBE=";
           };
 
+          # Apache Tika server 
+          tika-server = with final; stdenv.mkDerivation rec {
+            name = "tika-server-1.26";
+  
+            src = fetchurl {
+              url = https://archive.apache.org/dist/tika/tika-server-1.26.jar;
+              sha256 ="sha256-GLXsW4p/gKPOJTz5PF6l8DGjwXvIPoirDSmlFujnPZU=";
+            };
+  
+            dontUnpack = true;
+            buildInputs =with nixpkgs; [
+              jdk
+              tesseract
+              gdal
+              gnupg
+            ];
+            nativeBuildInputs = [
+              makeWrapper
+            ];
+            installPhase = ''
+            echo "Installing.. "
+              mkdir -pv $out/share/java $out/bin
+              ls -l ${src}
+            cp ${src} $out/share/java/tika-server-1.27.jar
+            makeWrapper ${jre}/bin/java $out/bin/tika-server \
+              --add-flags "-jar $out/share/java/tika-server-1.27.jar" \
+              --set _JAVA_OPTIONS '-Dawt.useSystemAAFontSettings=on' \
+              --set _JAVA_AWT_WM_NONREPARENTING 1
+              '';
+            meta = {
+              homepage = "https://tika.apache.org/";
```

Comment:
I'm not sure that is what they are running. it says `tika-extractor` and it runs on port 8081
https://github.com/ipfs-search/ipfs-search/blob/master/docker-compose.yml
I couldn't see what is inside their docker image. tika server runs on port  9998 by default. I don't think it's the same thing.

Reply:
ok. i might have made a mistake. will check please

Reply:
cool, let me know.
This is the one last piece before the ipfs-crawler can be run.